analyses/cell-contamination-removal-analysis/02-integrative-analysis.Rmd

```diff
@@ -0,0 +1,364 @@
+<div class="reporthead"><br/>
+**PI: `r params$PI_NAME`**  
+**Project: `r params$PROJECT_NAME`**  
+Task: `r params$TASK_ID`  
+Project Lead(s): `r params$PROJECT_LEAD_NAME`  
+Department: `r params$DEPARTMENT`  
+
+<br />  
+
+DNB Bioinformatics Core Analysis Team: 
+<br />  
+
+>**Lead Analyst(s): `r params$LEAD_ANALYSTS`**  
+>Group Lead: `r params$GROUP_LEAD`  
+<br />
+>**Contact E-mail:** `r params$CONTACT_EMAIL`  
+>**DNB Bioinformatics Core Pipeline:** `r params$PIPELINE`  
+
+Date started: `r params$START_DATE`  
+Date completed:  `r params$COMPLETION_DATE`  
+Report generated: `r format(Sys.time(), '%H:%M:%S %Z %m/%d/%Y')` \
+
+Reviewed by: _____________________   Date: ____________ \
+</div>
+\pagebreak
+  
+# Information about this notebook
+Biological heterogeneity in single-cell RNA-seq data is often confounded by technical factors including sequencing depth. The number of molecules detected in each cell can vary significantly between cells, even within the same celltype. Interpretation of scRNA-seq data requires effective pre-processing and normalization to remove this technical variability (for more information, see also [scRNA-seq Dataset Integration](https://www.singlecellcourse.org/scrna-seq-dataset-integration.html#seurat-v3-3-vs-5-10k-pbmc)).
+
+Hence, integration of single-cell sequencing datasets, for example across experimental batches, donors, or conditions, is often an important step in scRNA-seq workflows. Integrative analysis can help to match shared cell types and states across datasets, which can boost statistical power, and most importantly, facilitate accurate comparative analysis across datasets. Here, we include the following integration methods for integrative analysis:
+
+(1) [Seurat ‘anchor-based’](https://satijalab.org/seurat/articles/seurat5_integration) integration workflow by [Choudhary & Satija, 2022](https://doi.org/10.1186/s13059-021-02584-9),
+(2) [Harmony](https://portals.broadinstitute.org/harmony/) integration workflow by [Korsunsky et al., 2019](https://www.nature.com/articles/s41592-019-0619-0), and
+(3) [Liger](https://welch-lab.github.io/liger/articles/liger_with_seurat.html) integration workflow by [Welch et al., 2019](https://doi.org/10.1016/j.cell.2019.05.006).
+
+Different number of dimensions for different integration algorithms (Seurat, Harmony, Liger, etc.) is common and depending on the algorithm they come with their corresponding variables depending on how the algorithm is being used. Each of these integration algorithm takes into account typically the same variables but the math they use to transform them is so different that you can't use the same number of resolutions. Even though they use the same word, resolution doesn't actually effect each algorithm the same way and their maximums can be different. 30 for Seurat, 30 for Harmony and 20 for Liger give the most similar integration results when all other variables are taken into account while also not wasting too much compute time.
```

Comment:
I think this report should be updated to match the report generated in the `intergrative-analysis` module for clarity and consistency.

```suggestion
Different integration algorithms, such as Seurat, Harmony, and Liger, often use different numbers of dimensions. The optimal number of dimensions depends on the specific algorithm and its associated parameters. While these algorithms generally consider similar variables, the mathematical approaches they use to process and transform those variables differ significantly, which means the same number of dimensions (or "resolutions") doesn't work equally well across all algorithms.

Even though they all refer to "resolution," the effect of resolution can vary from one algorithm to another, and the maximum number of dimensions they can effectively use may differ. Based on our experience, we recommend using 30 dimensions for Seurat, 30 for Harmony, and 20 for Liger. These settings tend to produce the most consistent integration results without unnecessarily increasing computational load.
```

Reply:
Thank you for noticing this! The notebook has been updated here 7acd646a58e08fdcd2456571df2425774eaf0ead.